public/assets/css/style.css

```diff
@@ -686,23 +687,114 @@ input::placeholder {
                 max-width: 60%;
                 align-self: flex-start;
         }
+        
+
     }
 
     /* X-Large devices (large desktops, 1200px and up) */
     
     @media (min-width: 1200px) {
         /* *************** HEADER *************** */
+        .title{
+                font-size: 4em;
+                margin-bottom: 1rem;
+                margin-top: 1rem;
+        }
+        .title h1{
+                margin-bottom: 0;
+        }
+        .logo{
+                margin-left: 3rem
+        }
         /* *************** NAVBAR *************** */
+        #searchQueryInput{
+                width: 50rem;
+        }
+        .nav-title{
+                margin-right: unset;
+        }
         /* *************** SECTION LIST TITLE *************** */
+        .createCrime{
+                margin-top: 7rem;
+                margin-right: 0;
+        }
+        .listCrimes{
+                background:no-repeat url(../images/Carnet-anneaux.svg);
+                height: 540px;
+                margin-left:2rem;
+        }
+        .listCrimes h1{
+                margin: 1rem;
+                font-size: 40px;
+        }
+        .listCrimes a{
+                max-width: 80%;
+        }
         /* *************** SECTION CREATE CRIME *************** */
+        .formcrime{
+                background:no-repeat url(../images/Carnet-anneaux.svg);
+                padding-top: 10rem;
```

Comment:
no more than 4 rem for padding and margin, if it is higher you need to use flexbox or something else